packages/filecoin-address/src/index.ts

```diff
@@ -129,16 +136,23 @@ export function newBLSAddress(pubkey: Uint8Array): Address {
 }
 
 /**
- *
- * newDelegatedAddress returns a Delegated address
+ * newDelegatedAddress returns an address using the Delegated protocol.
  */
 export function newDelegatedAddress(
-  namespace: Uint8Array,
+  namespace: number,
```

Comment:
@Schwartz10 I think we should always pass namespace as a number, since it will always be an ID that can be represented as a normal number. The user shouldn't have to worry about converting that to the right size `Uint8Array`